master/buildbot/statistics/storage_backends/__init__.py

```diff
@@ -0,0 +1,19 @@
+# This file is part of Buildbot.  Buildbot is free software: you can
+# redistribute it and/or modify it under the terms of the GNU General Public
+# License as published by the Free Software Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright Buildbot Team Members
+
+from buildbot.statistics.storage_backends.base import StatsStorageBase
+from buildbot.statistics.storage_backends.influxdb_client import InfluxStorageService
+
+__all__ = ['StatsStorageBase', 'InfluxStorageService']
```

Comment:
So, should I remove everything from here and leave this file empty or just remove the `__all__`? I'm using it so pylint won't complain.